src/LuaInterface.cpp

```diff
@@ -592,20 +593,20 @@ void LuaInterface::renameVar(TVar* var)
     } else {
         newName.append(R"([")" + vars.back()->getNewName() + R"("])");
```

Comment:
The unmodified line 2 above this one should be:

    newName.append(QStringLiteral("[%1]").arg(QString::toNumber(vars.back()->getNewName())));

and this non-number form (what about `LUA_TBOOLEAN` type?):

    newName.append(QStringLiteral("[\"%1\"]").arg(QString::toNumber(vars.back()->getNewName())));